scripts/ama.coffee

```diff
@@ -18,6 +18,7 @@
 # Author:
 #   Gregory Cerna
 #   Steven Diaz
+#   Samuel Tallent
```

Comment:
Make sure to add your listWeights command to the comments at the top as well as in the ama help method